src/pyhf/schemas/1.0.0/model.json

```diff
@@ -3,7 +3,7 @@
     "$id": "https://scikit-hep.org/pyhf/schemas/1.0.0/model.json",
     "type": "object",
     "properties": {
-        "channels": { "type": "array", "items": {"$ref": "defs.json#/definitions/channel"} },
+        "channels": { "type": "array", "items": {"$ref": "defs.json#/definitions/channel"}, "minItems": 1 },
```

Comment:
As this changes the schema can we bump it to `1.0.1`?

Reply:
This actually doesn't change the schema. Empty channels was always going to break our code. Unless people are dealing with empty channels a lot, this is fixing a problem on our side.